### ISSUE TITLE
Bitcoin Core

### ISSUE THREAD

**Issue by DrDave517** [bitcoin-dot-org/bitcoin.org#1334]
I am running two bitcoin core nodes.

Neither one is working. I am getting "no block source available." And this, after many hours of synchronizing. I went to the forums and tried the solutions. Nothing is working. 

This leads me to believe that the core software is deeply flawed. Do you have a solution?

Reach me at nosema@mac.com.

David Haddad

**Comment by jonasschnelli** [comment]
Is your outbound port 8333 blocked?

**Comment by MarcoFalke** [comment]
Please upload your debug.log.

You can find it in your [data dir](https://en.bitcoin.it/wiki/Data_directory#Default_Location).

**Comment by DrDave517** [comment]
2016-07-22 03:39:31 SOCKS5 connecting 68.48.214.241
2016-07-22 03:39:51 ERROR: Error reading proxy response
2016-07-22 03:39:51 SOCKS5 connecting seed.bitcoinstats.com
2016-07-22 03:53:36 ERROR: Error reading proxy response
2016-07-22 03:53:36 SOCKS5 connecting 119.81.178.116
2016-07-22 03:54:11 ERROR: Error reading proxy response
2016-07-22 03:54:11 SOCKS5 connecting bitseed.xf2.org
2016-07-22 03:54:34 ERROR: Error reading proxy response
2016-07-22 03:54:34 SOCKS5 connecting 173.44.55.46
2016-07-22 03:54:54 ERROR: Error reading proxy response
2016-07-22 03:54:54 SOCKS5 connecting seed.bitcoin.jonasschnelli.ch
2016-07-22 03:55:14 ERROR: Error reading proxy response
2016-07-22 03:55:15 SOCKS5 connecting 209.208.110.92
2016-07-22 03:55:35 ERROR: Error reading proxy response
2016-07-22 03:55:35 SOCKS5 connecting seed.bitcoin.sipa.be
2016-07-22 04:16:59 ERROR: Error reading proxy response
2016-07-22 04:16:59 SOCKS5 connecting 185.146.168.253
2016-07-22 04:17:19 ERROR: Error reading proxy response
2016-07-22 04:17:19 SOCKS5 connecting dnsseed.bluematt.me
2016-07-22 04:17:39 ERROR: Error reading proxy response
2016-07-22 04:17:40 SOCKS5 connecting 167.160.36.52
2016-07-22 04:18:00 ERROR: Error reading proxy response
2016-07-22 04:18:00 SOCKS5 connecting dnsseed.bitcoin.dashjr.org
2016-07-22 04:18:20 ERROR: Error reading proxy response
2016-07-22 04:18:20 SOCKS5 connecting 195.154.114.179
2016-07-22 04:18:40 ERROR: Error reading proxy response
2016-07-22 04:18:40 SOCKS5 connecting seed.bitcoinstats.com
2016-07-22 15:10:23 ERROR: Error reading proxy response
2016-07-22 15:10:23 SOCKS5 connecting 82.145.59.7
2016-07-22 15:10:43 ERROR: Error reading proxy response
2016-07-22 15:10:43 SOCKS5 connecting bitseed.xf2.org
2016-07-22 15:11:03 ERROR: Error reading proxy response
2016-07-22 15:11:04 SOCKS5 connecting 85.228.201.80
2016-07-22 15:11:24 ERROR: Error reading proxy response
2016-07-22 15:11:24 SOCKS5 connecting seed.bitcoin.jonasschnelli.ch
2016-07-22 15:11:44 ERROR: Error reading proxy response
2016-07-22 15:11:44 SOCKS5 connecting 82.225.196.183
2016-07-22 15:12:04 ERROR: Error reading proxy response
2016-07-22 15:12:04 SOCKS5 connecting seed.bitcoin.sipa.be
2016-07-22 15:12:24 ERROR: Error reading proxy response
2016-07-22 15:12:25 SOCKS5 connecting 192.237.202.118
2016-07-22 15:12:45 ERROR: Error reading proxy response
2016-07-22 15:12:45 SOCKS5 connecting dnsseed.bluematt.me
2016-07-22 15:13:05 ERROR: Error reading proxy response
2016-07-22 15:13:05 SOCKS5 connecting 52.39.187.3
2016-07-22 15:13:25 ERROR: Error reading proxy response
2016-07-22 15:13:25 SOCKS5 connecting dnsseed.bitcoin.dashjr.org
2016-07-22 15:13:45 ERROR: Error reading proxy response
2016-07-22 15:13:46 SOCKS5 connecting 212.90.156.93
 

BTW, I started with a default port, I think it was 9050. But on further reading I switched it to 8333. After the freeze up, I went to 9150, 9050, 8333, restarting the core each time. Same result.
